viewer/packages/rendering/src/glsl/sector/primitives/cone.frag

```diff
@@ -156,7 +157,6 @@ void main() {
       }
   #endif
 
-
-    float fragDepth = updateFragmentDepth(p, projectionMatrix);
-    updateFragmentColor(renderMode, color, v_treeIndex, normal, fragDepth, matCapTexture, GeometryType.Primitive);
+    updateFragmentDepth(p, v_projectionMatrix);
+    updateFragmentColor(renderMode, color, v_treeIndex, normal, gl_FragCoord.z, matCapTexture, GeometryType.Primitive);
```

Comment:
Why use gl_FragCoord.z?

Reply:
This is wrong, I think, I pushed while testing/identifying the issue, my bad!